.eslintrc.js

```diff
@@ -178,7 +178,7 @@ module.exports = {
         "prefer-const": "error",
         "prefer-reflect": "off",
         "prefer-rest-params": "off",
-        "prefer-spread": "error",
+        "prefer-spread": "off",
```

Comment:
`eslint` demands not to use `apply`, instead it wants spread operator. But actually we can't do this, while not using es6.